### PR TITLE
TT-3 Added components for story list, story item and viewable stories

### PR DESCRIPTION
**Pull request by sanatdeshpande1**
Hi team, 

I have added components for story list, story item (story-item.js) and viewable stories (view-story.js). I have also added the functionality where a story automatically closes after 15 seconds of time interval. I have not added the timer for it though, since I thought it would be better if we can discuss regarding the position of the timer on the UI.

This is how the story list looks like:

image

This is how the viewable story component looks like:

image

### CODE REVIEW

File: src/services/story-service.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ */
+export const findAllStories = () => {
+  // Can use redux
+  console.log("GET request for fetching all stories associated with the user.");
+  return stories;
+};
+
+/**
+ * Finds and returns the story object associated with the provided id.
+ * @param {Number} sid ID of the story to be searched and returned from database.
```
Comment (sghosalkar) [inline src/services/story-service.js:15]: I think `sid` will be string

File: src/components/tuiter/index.js [hunk 2]
```diff
@@ -1,53 +1,58 @@
 import React from "react";
 import Navigation from "../navigation";
 import WhatsHappening from "../whats-happening";
-import {Routes, Route, HashRouter} from "react-router-dom";
+import { Routes, Route, BrowserRouter, HashRouter } from "react-router-dom";
```
Comment (sghosalkar) [inline src/components/tuiter/index.js]: We can remove `HashRouter` from imports

**Review by sghosalkar — APPROVED** [review]
Looks good!

**Review by galen94 — APPROVED** [review]

**Review by galen94 — APPROVED** [review]
This looks great! I don't see any other issues other than what was already noted.

**Pull request merged**